requirements/local.txt.sample

```diff
@@ -0,0 +1,16 @@
+# This file is meant to be customized for personal development environments.
+# Feel free make a copy without the .sample extension to use locally.
+#
+# requirements/local.txt is in .gitignore so you will not be prompted to (nor
+# should you) commit your custom version of that file.
+#
+# After adding your custom requirements you can run one simple command to keep
+# your virtualenv up to date:
+#
+#   pip-sync requirements/local.txt [--dry-run]
+
+-r dev-requirements.txt
+
+django-debug-toolbar
```

Comment:
nit: do we also want to comment these out and leave it for the dev to uncomment the ones they want?

Reply:
I decided not to comment them out since they seem like reasonable recommendations, and it makes this file more useful right out of the box, even if you don't have anything custom to add.